### ISSUE TITLE
PM Meeting Agenda

### ISSUE THREAD

**Issue by kcoronel** [hackforla/ballotnav#461]
### Overview
Meeting Agendas for BallotNav PM meetings

#### To create a new agenda
- [ ] Copy the template below into a new comment
- [ ] Edit the text you pasted with new items for your agenda
- [ ] Add the link to the new comment with its date under Resources/Instructions

### Resources/Instructions
The prework (if any)should be done before the meeting so that it can be discussed during the meeting.  It usually takes about 5 min, if your team reaches all issues being labeled and milestones setup.  During cleanup it typically takes about 30 min weekly.

#### Template for next meeting agenda
```
## [Date] Agenda  and notes

### Prework
- [ ] issue for audit [to be added once there is an issue]

### Roll Call

#### Recurring items: Happens on the PM Sunday meeting
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/admin/issues/assigned/ExperimentsInHonesty)
       - [ ] [Karen](https://github.com/hackforla/admin/issues/assigned/kcoronel)
 - [ ] continue adding labels to the issue that are still missing them (see audit links above)

#### New Items
- [ ] 
- [ ] Make new agenda for next week
  - [ ] Carry forward the "New Items" that we did not get to look at today

#### FYIs

#### Notes from meeting

#### Items for next week's agenda
```

#### Meetings Past and future
[2022-01-08](https://github.com/hackforla/admin/issues/38#issuecomment-998089812)

**Comment by kcoronel** [comment]
## 2/19/22 Agenda  and notes

### Prework
- [ ] issue for audit [to be added once there is an issue]

### Roll Call
Karen 
Stacey 
Layne 
Gayathri

#### Recurring items: Happens on the PM meeting
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Karen](https://github.com/hackforla/admin/issues/assigned/kcoronel)
       - [ ] Stacey (https://github.com/hackforla/ballotnav/issues?q=assignee%3Astaceyrebekahscott+is%3Aopen)
      - [ ] Layne (https://github.com/hackforla/ballotnav/issues?q=is%3Aissue+is%3Aopen+layneam)
      - [ ] Gayathri (https://github.com/hackforla/ballotnav/issues?q=assignee%3Ayoursgayathri+is%3Aopen)

#### New Items
- [ ] 
- [ ] Make new agenda for next week
  - [ ] Carry forward the "New Items" that we did not get to look at today

#### FYIs

#### Notes from meeting

#### Items for next week's agenda

**Comment by kcoronel** [comment]
## 2/19/22 Agenda and Notes

### Prework
None this time.  

### Roll Call
- [ ] Bonnie 
- [ ]  Stacey 
- [ ]  Layne 
- [ ] Gayathri 

#### Recurring items: 
This is the first meeting as a team. 

#### New Items
- [ ] Intros
- [ ] MVP document review 
- [ ] Chat with Bonnie on our agreed upon next steps
    - [ ] Developing our north star for project 
    - [ ] Developing our OKRs a
    - [ ] Developing branding and product fit
    - [ ] Designing a better onboarding process and wiki page for the team 
- [ ] Make new agenda for next week
  - [ ] Carry forward the "New Items" that we did not get to look at today

#### Notes from meeting
**MVP Document** 
After reviewing the MVP document with Bonnie, she gave us the following suggestions 
1. **Build a flexible interface** for updating the location data that is easy to use. 
- create a heighrghy of who can validate ex: state/county/jurisdiction says here's a list of the counties and the authorized users, here's their email address, create accounts for them and then we onboard them (welcome email, FAQ, etc.) 
- interface can be modeled to the rules/laws around jurisdiction ex: Jurisdiction A only allows you to drop off 

This is Bonnie's hypothesis on why updating election information may be difficult for a jurisdiction. This hypothesis would need to be validated via user research. 1. A website to put the ballot/election information on it that is easy to update which most government sites are not 2. An interface where someone non-technical can enter data. If you spread the labor to last mile, everyone is doing less work and updates will be made faster. 

2.  Dependant way to jur

2. other people to put data (volunteers) have to have interface and a way to validate data [remove] 
2. Data Security. An example for how to go on about building this is from Amazon who uses mechanical turf- where you can part out work to people and what you do is overlap work so that multiple people are checking the same thing. That enables you to validate a person's listing. EX: If a person's listings are marked as 'BAD' (updates don't match), then you would remove all of their data. 
3. 

Jurisdiction can put in data ; if volunteers put in - need interface and way to validate data
Jurisdiction will still need back end interface
Hierarchy - list of counties and authorized users - email address exist - set up account over listtings



#### Items for next week's agenda



Flexible interface for county
What kind of dropbox is this and then builds profile for them
All data should roll to-us- api

V1: renting it to jurisdiction- cloud based= we take care of security and data

Service to input data themselves - charge t input data
Send user interface research

SLA issue

Board of advisors - partner with
Problem wE ARE TRYING TO SOLVE AND HOW TO BEST SOLVE PRO
[VOTE.ORG](http://vote.org/)
BLACK VOTERS

PEOPLE WHO ARE PRO VOTER - EASY AS POSSIBLE FOR BETTER TO VOTE

CONTACT INFO DFOR

REACH OUT TO GIGI

RESEARCH PROGRAM - VALIDATION AND RECORD - ANOTHER ROUND OF CHANGES

2023 - UPDATE MVP

SET UP BOARD OF ADVISORS - WHICH JURISDI- DO IT WITH THEM - SEE HOW IT WORKS - INTEGRATE - AFTER MIDTERMS ARE OVER- BACK TO BOARD - WHAT TO DO

BONNIE TO CONNEXT US WITH YOUTH JUSTICE FOLKS FOR HAVING ADVISORY BOARD

LIST OF THINGS THAT MAY BE OF VALUE

Items for next week's agenda
1PASSWORD
tECHNICAL

Edit roster

Dashboard org figma

offboard and onboard

**Comment by layneam** [comment]
Suggestions for "create a list of things that may be of value to stakeholders as we talk to them", to discuss @ PM meeting
- Mutually beneficial introductions
- Lessons learned, mistakes, professional experiences
- Business resources—an article/book, a conference, a new piece of software, or any other professional tool 
- Offer your unique talents, strengths, or subject matter expertise
- Give a referral

**Comment by kcoronel** [comment]
## 2/28/22 Agenda  and notes

### Prework
- [ ] #456 [wiki page draft](https://docs.google.com/document/d/1bjVlgVfbzYSWi--ZJ1MdvqT3YIG8UP-H/edit?rtpof=true&sd=true)
- [ ] #13 
- [ ] #451

### Roll Call
Stacey 
Gayathri 
Layne 
Punit 
Karen

#### Recurring items: Happens on the PM Sunday meeting
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Gayathri] 
       - [ ] Stacey 
       - [ ] Layne 
       - [ ] [Karen](https://github.com/hackforla/admin/issues/assigned/kcoronel)
 - [ ] continue adding labels to the issue that are still missing them (see audit links above)

#### New Items
- [ ] 
- [ ] Make new agenda for next week
  - [ ] Carry forward the "New Items" that we did not get to look at today

#### FYIs

#### Notes from meeting

#### Items for next week's agenda

**Comment by kcoronel** [comment]
## 3/7/21 Agenda  and notes

### Prework


### Roll Call


#### Recurring items: Happens on the PM Sunday meeting
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Gayathri](https://github.com/hackforla/ballotnav/issues?q=assignee%3Ayoursgayathri+is%3Aopen)
       - [ ] [Stacey](https://github.com/hackforla/ballotnav/issues?q=assignee%3Astaceyrebekahscott+is%3Aopen)
       - [ ] [Layne](https://github.com/hackforla/ballotnav/issues?q=assignee%3Alayneam+is%3Aopen)
       - [ ] [Karen](https://github.com/hackforla/admin/issues/assigned/kcoronel)
       - [ ] Punit 

#### New Items
- [ ] 
- [ ] Make new agenda for next week
  - [ ] Carry forward the "New Items" that we did not get to look at today

#### FYIs

#### Notes from meeting

#### Items for next week's agenda

**Comment by kcoronel** [comment]
## 3/14/21 Agenda and notes

### Prework


### Roll Call
Stacey
Gayathri
Karen


#### Recurring items: Happens on the PM Sunday meeting
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Gayathri](https://github.com/hackforla/ballotnav/issues?q=assignee%3Ayoursgayathri+is%3Aopen)
       - [ ] [Stacey](https://github.com/hackforla/ballotnav/issues?q=assignee%3Astaceyrebekahscott+is%3Aopen)
       - [ ] [Layne](https://github.com/hackforla/ballotnav/issues?q=assignee%3Alayneam+is%3Aopen)
       - [ ] [Karen](https://github.com/hackforla/admin/issues/assigned/kcoronel)
       - [ ] Punit 

#### New Items
- [ ] Scheduling meeting with Drew for background on project 
- [ ] Updates on UI/UX job post 
- [ ] Review this issue #469 anc complete by Wednesday 3/9
- [ ] Creating a two-week sprint for the external docs we want to get done quickly/prioritize 
     - [ ] MVP Document 
     - [ ] 1-pager 
     - [ ] About us deck? 
     - [ ] Roadmap? 
     - [ ] ?
- [ ] Make new agenda for next week
  - [ ] Carry forward the "New Items" that we did not get to look at today

#### FYIs

#### Notes from meeting


#### Items for next week's agenda

**Comment by kcoronel** [comment]
## [Date] Agenda  and notes

### Prework
- [ ] issue for audit [to be added once there is an issue]

### Roll Call

#### Recurring items: Happens on the PM Sunday meeting
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/admin/issues/assigned/ExperimentsInHonesty)
       - [ ] [Karen](https://github.com/hackforla/admin/issues/assigned/kcoronel)
 - [ ] continue adding labels to the issue that are still missing them (see audit links above)

#### New Items
- [ ] 
- [ ] Make new agenda for next week
  - [ ] Carry forward the "New Items" that we did not get to look at today

#### FYIs

#### Notes from meeting

#### Items for next week's agenda

**Comment by ExperimentsInHonesty** [comment]
Please add to next agenda, doing a cleanup of old agenda issues
- https://github.com/hackforla/ballotnav/issues/437
- https://github.com/hackforla/ballotnav/issues/431

**Comment by kcoronel** [comment]
## 3/28 Agenda  and notes

### Prework
- Kanban methodologies 

### Roll Call
Stacey 
Layne 
Gayathri 
Karen

#### Recurring items: Happens on the PM Monday meeting 
 - [ ] Accountability and Support Check.
    - [ ] Review assignments for each PM
        - [ ] [Gayathri](https://github.com/hackforla/ballotnav/issues?q=assignee%3Ayoursgayathri+is%3Aopen)
       - [ ] [Stacey](https://github.com/hackforla/ballotnav/issues?q=assignee%3Astaceyrebekahscott+is%3Aopen)
             - Stacey creating new project boards for UI/UX research 
       - [ ] [Layne](https://github.com/hackforla/ballotnav/issues?q=assignee%3Alayneam+is%3Aopen) 
             - Layne, Stacey and I are working on an onboarding issue, taking inspiration from [here ](https://github.com/hackforla/website/issues/new?assignees=&labels=Feature%3A+Onboarding%2FContributing.md%2C+good+first+issue%2C+role%3A+user+research&template=pre-work-template---ux.md&title=Pre-work+Checklist%3A+Researcher%3A+%5Breplace+brackets+with+your+name%5D)
       - [ ] [Karen](https://github.com/hackforla/ballotnav/issues/assigned/kcoronel)
             - Create a new issue for our Kanban cleanup 
  
#### New Items
ON PAUSE ~- [ ] Creating a two-week sprint for the external docs we want to get done quickly/prioritize 
          - [ ] MVP Document (Gayathri) 
          - [ ] 1-pager (Karen) 
          - [ ] About us deck (Karen & Layne) 
          - [ ] Roadmap~ 
- [ ] Onboarding new members
      - create issue for members and have them assigned to it 
- [ ] Make new agenda for next week
  - [ ] Carry forward the "New Items" that we did not get to look at today
     - [ ] Finalize PM assignments
     - [ ]  Go through PM workflow process for Kanban (Go over criteria for clean-up)

#### FYIs

#### Notes from meeting

#### Items for next week's agenda

**Comment by staceyrebekahscott** [comment]
## [04/04/22] Agenda  and notes

Accountability and Support Check.
Update on current progress for each PM

Stacey:

Created project for Market Fit Research. I am adding new issues and working on a project charter/plan. 

As per our UX meeting on Tuesday, the UX team have been adding to a doc in Drive called "Market Fit Research brainstorming". The goal for this week was to continue adding to this doc, expanding on the topics of research techniques, partnership organizations, competitive analysis of direct competitors and existing research on the state of voting via ballot drop off. The team has been adding to this doc all this week. All of these will be issues in the project.
 
I have also encouraged discussion in the ballot nav design channel in Slack. All updates will take place there and in the issues in lieu of a UX meeting on Monday. Rather than following the regular format of the accountability check during the team meeting, I'd like to present a short update and open it up to the other team members to add what they have worked on, either verbally or via the Zoom chat. I'd like to see if this is a more productive (and interesting) way to run the meetings and get updates from members. 

My goal this week is also to organize a summary of the status of the Board of Advisors project. I want to summarize the call with Jyoti Nanda, and write up a project plan. Karen said she would share the current status of the BallotNav project with Bonnie, and I will send her the summary of the Board of Advisors project so she knows it was not dropped completely. Either she will be OK with us picking it back up again once the Market Fit Research is done, or if she decides she wants another team to work on it, it will be in a form that it can be transitioned to another team. 



Karen:




Layne:




Gayathri:       
       


#### New Items

#### FYIs

#### Notes from meeting

#### Items for next week's agenda

**Comment by staceyrebekahscott** [comment]
## [04/18/22] Agenda  and notes

### Roll Call
 Accountability and Support Check.
 Update on current progress for each PM

Stacey:
- Created the project charter for the Market Fit Research via a Miro board: [Market Fit Research in Miro](https://miro.com/app/board/uXjVO9_TkJE=/)
- Reviewed issue #486 , read linked articles in literature review doc and PDFs. Awaiting slideshow from Carrie and Lihui documenting the analysis. 
- Reviewed issue #488 , read through slideshow. Slideshow is located in the Shared Folder for BallotNav. I advised Federica and Seeyoung to move it to the proper location in the Shared Drive, and to connect with Karen to make sure they have proper access to the Drive. 
- Followed up with Zuri re: issue #487 
- Followed up with Jon re: issue #481 
- Created new issue, #489 . This issue is to access the capability of ongoing support of the project by HfLA. Will be seeking the assistance of the other PMs on this issue. 

Karen:

Layne:

Gayathri:







#### Notes from meeting

#### Items for next week's agenda

**Comment by staceyrebekahscott** [comment]
## [05/02/22] Agenda  and notes

### Roll Call & Accountability Check
Stacey:
- Moved #435 to Ice Box in BN2.0, added comment
- Moved #442 to Prioritized backlog in BN2.0, added comment
- Updated #479 in BN2.0 for Lihui Zhang- Need to off board her from the team, see details in comment
- Moved #13 to Market Fit Research (MFR), added comment
- Moved #463 to MFR, added comment
- Moved #478 to MFR, added comment
- Followed up on #488 , #487 and #481 
- Completed drafts of current status of MFR and BN2.0, sent message to PMs requesting feedback via #483 

Karen:

Layne:

Gayathri: 

#### New Items

#### FYIs

#### Items for next week's agenda

**Comment by staceyrebekahscott** [comment]
Notes from PM Meeting 06/21/22:

- Using the Wiki page more effectively to organize- To serve as a general framework of what we have done so far. 
  - Creating a Wiki to document the project progress- what BallotNav 2.0 accomplished, why we are doing Market Fit Research now (this could be a replacement for the on boarding slide deck) 
  - Wiki can serve as a central place to find everything and document it- the process of how we come to decisions, what our research process is (possible replacement for Research Methodologies slide deck). 
  - Wiki as a table of contents for the Google Drive- one central place to find key documents. 
  
- Need to recruit more UX people. 
  - Start with finding someone who can help with documentation and can advise on planning: #496 

To Do: 
- [ ] Look at examples or templates of Wiki Pages for best practices, ideas
- [ ] start draft of what we want in ours (Ex: headings for Approach, MVP Testing)

**Comment by staceyrebekahscott** [comment]
Notes from Async meeting on 06/27/22:

Layne: 

- I'm still planning to attend organizing meetings this week and next.
- I attended a couple at the end of last week which didn't lead to anything very productive, which is fine. I was signed up for a 3 hour webinar on Sunday (Muslims Vote) but it was cancelled without explanation. The useful ones I attended last week are summarised on the issue, #492 
- This week I have a listening session for Native Voices and a We Are All Organizers meeting put on by the Democratic party, same folks as last week, and a letter writing event on Saturday.
- Next week I have a PA voter registration event. For this week and next I'll be looking for other events I can attend.
- The following week I have a voting rights event and 9 hours of Indivisible group leader events.
- I will start a mind map for the customer needs we're discovering and bring it to the group meeting happening tomorrow.
- Reflection: I think these events are fine, and as we gather more data I am building a list of people I'm meeting who I can possibly tap to look at any ideas or prototypes we come up with in the future. 
- At this point I would like to see what customer interviews will yield, I think we would accelerate our learning with both types of information coming in. 

Stacey:

- I assigned Zuri to #494  and once she sets up her access to 1Password and the BallotNav email, she can start sending out more emails to orgs to schedule interviews. 
- I added notes to #494  from communication I had with Mya Stark. She has long COVID and has a hard time speaking on the phone, so I created a doc for her to respond to. She said she does not have experience with voting orgs, but that she came up with the idea for BallotNav from the point of view of a voter who could not find information. So her feedback will be more useful once we have a prototype or MVP to test.
- Posted for the UX Researcher/ Designer in UI-UX, General and New Joiner Orientation, #496 

Karen:
-Posted UX Researcher/ Designer on UI-UX CoP

**Comment by ExperimentsInHonesty** [comment]
Copied this from project board during GitHub Project Board migration of 2024-08.

> Timeline discussed with Bonnie at our4/9 meeting. 
> 1. Improve validation interface between now and midterm elections 
> 2. Ask for funding and partnership with non-partisan organizations or organizations within the voting, democracy, data science, social equity domain. Advocate for funding for security/penetration testing. 
> 3. Product is ready by April 2022, giving us time to promote
